app/templates/play/ladder/ladder.jade

```diff
@@ -168,7 +176,7 @@ block content
       li
         a(href="#winners", data-toggle="tab", data-i18n="ladder.winners") Winners
 
-  div.tab-content
+  div.tab-content(style="display:"+view.displayTabContent) //in order to control page height
```

Comment:
I think this would be clearer if the entire string was in the variable, like so: 

LadderView.coffee
```
...
@displayTabContent = 'display: block'
...
@displayTabContent = 'display: none'
...
```

And then the template could just be this: 

ladder.jade
```
...
div.tab-content(style=view.displayTabContent)
...
```

Also, I think the comment could be more clear if it described why it is a conditional showing of the content, for example saying `// only render content when tournament is active` or something like that.